docs/example-api-calls/google-workspace.md

```diff
@@ -1,235 +1,209 @@
 # API Call Examples for Google Workspace
 
-Example commands that you can use to validate proxy behavior against the Google Workspace APIs.
+Example commands (*) that you can use to validate proxy behavior against the Google Workspace APIs.
 Follow the steps and change the values to match your configuration when needed.
 
-To use, ensure you've set env variables on your machine:
-```shell
-# no slash at the end, and no function or lambda, just the base url of the deployment
-export PSOXY_BASE=https://YOUR_PSOXY_HOST
-export PSOXY_USER_TO_IMPERSONATE=you@acme.com
-```
+For GCP, you can use the `-i` flag to impersonate the desired user identity option when running the testing tool. Example:
 
-For GCP, use
 ```shell
-export OPTIONS="-g -i $PSOXY_USER_TO_IMPERSONATE"
+node tools/test-psoxy.js -u [your_psoxy_url]/psoxy-gcal/calendar/v3/calendars/primary -i you@acme.com
 ```
 
-For AWS, change the role to impersonate with one with sufficient permissions to call the proxy
+For AWS, change the role to impersonate with one with sufficient permissions to call the proxy (`-r` flag). Example:
```

Comment:
```suggestion
For AWS, change the role to assume with one with sufficient permissions to call the proxy (`-r` flag). Example:
```

not your change, but "assume" is actually more in keeping with AWS's conventions